SrfGen/srfinfo2vm.py

```diff
@@ -2,7 +2,7 @@
 """
 REQUIREMENTS:
 PATH contains NZVM binary (github:ucgmsim/Velocity-Model/NZVM)
-PYTHONPATH contains Bradley_2010_Sa and Afshari_Stewart_2016_Ds
+PYTHONPATH contains Bradley_2013_Sa and Afshari_Stewart_2016_Ds
```

Comment:
Fix this comment - what does actually need to be in PYTHONPATH for this to work?